pasystem/pasystem-tool/tool/src/webapp/stylesheets/pasystem.css

```diff
@@ -75,38 +75,40 @@
   -o-transition: background-color 0.2s, color 0.2s;
   transition: background-color 0.2s, color 0.2s;
 
-  background-color: #f7e084;
-  color: #000 !important;
+  background-color: var(--bs-warning, #f7e084);
+  color: var(--sakai-brand--darker-7) !important;
   text-align: center;
-  padding: 6px 8px;
+  padding: 10px 9px;
   border-radius: 3px;
   text-decoration: none;
+  line-height: 0;
 }
 .pasystem-banner-alert-toggle-list-item {
-  position: absolute;
-  left: 50%;
-  transform: translateX(-50%);
-  top: 13px;
-  border: 1px solid #E0E0E0;
-  z-index: 1000;
+  background-color: var(--bs-warning, #f7e084);
```

Comment:
Will these colours be okay in dark mode?

Reply:
On dark mode:

![image](https://user-images.githubusercontent.com/15141743/203925145-86db9168-d9c7-40ca-9198-9988006a2a6e.png)